IntroComponentWithSign-upForm/script.js

```diff
@@ -0,0 +1,67 @@
+const form  = document.getElementById("form")
+const firstName  = document.getElementById("firstName")
+const lastName  = document.getElementById("lastName")
+const email  = document.getElementById("email")
+const password  = document.getElementById("password")
+//show error message
+const showErrorMessage = ( input , message ) => {
+    const span = document.querySelector("."+input.id)  
+    console.log(span.innerText = message);
+}
+// Show input success message
+const showSuccessMessage = ( input ) => {
+    const span = document.querySelector("."+input.id)
+    span.innerText = "";
+}
+//get field name
+const getFieldName = (input) =>{
+    return input.name;   
+} 
+// check required fields
+const checkRequired = (inputArr) => {    
+    inputArr.forEach( (input)=>{ 
+        if ( input.value.trim() === "")
```

Comment:
I'd move this into a variable, just because you are already applying a method on the value, but also, if you want to change this in the future, this would be easier to understand!

```suggestion
	const value = input.value.trim();
        if (value === "")
```